src/taskscontroller.py

```diff
@@ -0,0 +1,87 @@
+from logging import getLogger
+from typing import Tuple, List, Union
+
+from apscheduler.schedulers.blocking import BlockingScheduler
+from apscheduler.triggers.cron import CronTrigger
+import yaml
+
+from src.tasks import TaskWrapper
+from config.settings import *
+
+logger = getLogger("general")
+scheduler = BlockingScheduler()
+scheduler.configure(**SCHEDULER)
+
+
+class TasksController:
+    """A class that parses a configuration file,
+    sets tasks to be executed using apscheduler"""
+
+    def get_tasks(self) -> Union[List[Tuple[TaskWrapper, Dict]], None]:
```

Comment:
`get_tasks` return `List` of `Tuple[TaskWrapper, Dict]`. But semantically it should be a `List of Tasks`. Maybe to create a class or type or dict for this kind of structure?

it is difficult to understand (why tuple? What type is the first element, etc):
```
 tasks.append((TaskWrapper(**src), schedule))
```